packages/graphql/src/modules/user/app/authenticateFacebookUser.ts

```diff
@@ -59,10 +60,9 @@ const authenticateFacebookUser = ({
         providerId.providedBy === 'facebook' ? ({id: fbID, providedBy: 'facebook'} as ProvidersId) : providerId,
       );
 
-      await userRepository.updateUser({
-        id: existingUser.id,
-        proprieties: {providersIds: updatedProvidersIds},
-      });
+      existingUser.providersIds = updatedProvidersIds;
```

Comment:
Same here. If you have this kind of assignment em other places of your code, would be great to improve that.